Source/WebCore/Modules/url-pattern/URLPatternInit.h

```diff
@@ -25,8 +25,6 @@
 
 #pragma once
 
-#include <wtf/text/WTFString.h>
```

Comment:
Why remove this? Seems useful given the structure below.

Reply:
Seemed to build without it so I assumed that somehow the build system was able to link String class to this header file. I'll add it back.